docs/guides/connecting_to_your_data/components/where_to_run_code.md

```diff
@@ -0,0 +1,37 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Get an environment to run the code in this guide. Please choose an option below.
+
+<Tabs
+  defaultValue='cli'
+  values={[
+  {label: 'CLI', value:'cli'},
+  {label: 'YAML', value:'yaml'},
+  {label: 'python', value:'python'},
+  ]}>
+  <TabItem value="cli">
+
+If you use the Great Expectations CLI, run this command to automatically generate a pre-configured jupyter notebook:
+
+```console
+great_expectations --v3-api datasource new
+```
+
```

Comment:
Since `CLI` is its own tab here, I think one more sentence saying something like : "running the CLI command will open up a separate Notebook, and you can follow along in the YAML workflow below" would help users get oriented. Also with the added bonus that `YAML` is the default tab that is selected, the guide will read seamlessly :)